master/static/srv/get_logs.sql

```diff
@@ -0,0 +1,8 @@
+SELECT
+    trial_logs.id as id,
+    trials.state as state,
+    encode(message, 'escape') as message
+FROM trial_logs JOIN trials
+ON trial_logs.trial_id = trials.id
+WHERE trials.id = $1 AND trial_logs.id > $2
+ORDER BY trial_logs.id ASC;
```

Comment:
The CLI currently does this to print all the logs of the trial before moving on to following the logs.

Reply:
That doesn't seem great but if you would like to just make this PR 1 for 1 with the existing implementation I'd say we should at least file a ticket to use skip/limit for the logs. For a long running experiment this could be a lot of data.

Reply:
I prefer to maintain the existing implementation at the moment, but we should consider fixing this as part of the API rewrite project.